test/runtime/kvstore.go

```diff
@@ -87,12 +90,14 @@ var _ = Describe("RuntimeKVStoreTest", func() {
 	It("Etcd KVStore", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
+		By("Starting Cilium using etcd as backing key-value store")
 		vm.ExecContext(
 			ctx,
-			"sudo cilium-agent --kvstore etcd --kvstore-opt etcd.address=127.0.0.1:4001")
+			"sudo cilium-agent --kvstore etcd --kvstore-opt etcd.address=127.0.0.1:9732")
```

Comment:
Out of curiosity, why not `:2379`? Also, is the `--debug` flag omitted by accident? Just asking because it's set for consul (line 77)